Formula/git-hooks-go.rb

```diff
@@ -0,0 +1,22 @@
+class GitHooksGo < Formula
+  desc "Git hooks manager"
+  homepage "https://git-hooks.github.io/git-hooks"
+  url "https://github.com/git-hooks/git-hooks/archive/v1.3.0.tar.gz"
+  sha256 "518eadf3229d9db16d603290634af8ae66461ec021edf646e8bca49deee81850"
+  license "MIT"
+  head "https://github.com/git-hooks/git-hooks.git"
+
+  depends_on "go" => :build
+
+  conflicts_with "git-hooks", :because => "both install `git-hooks` binaries"
```

Comment:
As `git-hooks` deprecated, why dont we just rewrite the original formula with the golang version?

Reply:
Assuming the golang edition is the same functionality as the old one.

Reply:
Yeah I don't see why not, how would we go about doing that? (how many commits, what to put for commit messages, etc)

Reply:
No the functionality is different

Reply:
Yeah, I had this conversation with Samford before. The two are different so we can't overwrite them.

Reply:
I see, sgtm.